src/Logger/SimpleHttpLogger.php

```diff
@@ -71,7 +71,9 @@ public function trace($spans)
     protected function validResponse($headers)
     {
         foreach ($headers as $header) {
-            if (preg_match('/202/', $header)) {
+            // We want to match the status header (e.g. "HTTP/1.1 202 Accepted") but not any other
+            // headers that might have the substring "202"
+            if (preg_match('/^HTTP\/\S+\s+202/', $header)) {
```

Comment:
This will match linebreaks. Just use `.` character. It should match it on the same line. That's the RFC Spec https://tools.ietf.org/html/rfc2616#section-6

Reply:
Example here: 
[regexr.com/5mrc8](regexr.com/5mrc8) : this is matching
[regexr.com/5mrch](regexr.com/5mrch) : this is not matching
In your case, second matches [regexr.com/5mrck](regexr.com/5mrck)

Reply:
The spec you linked shows they're delineated by newlines, and PHP has already segmented them into an array. Therefore it's not possible for a linebreak to be present in one of these strings. So the example in https://regexr.com/5mrch cannot ever happen. Is this concern a blocking one, or perhaps more academic?

My concern with your suggestion is that regexes are hard enough to visually parse as-is, without deviating from the more common patterns. I'd say `\s` and `\S` are seen more often than `\h` and so are more likely to be recognized and understood, without the need to go look them up. I.e. I'm prioritizing code clarity over technical correctness, since in practice it won't ever manifest as a bug.